partiql-eval/src/main/kotlin/org/partiql/eval/internal/operator/rex/ExprError.kt

```diff
@@ -2,11 +2,12 @@ package org.partiql.eval.internal.operator.rex
 
 import org.partiql.eval.Environment
 import org.partiql.eval.ExprValue
-import org.partiql.spi.errors.TypeCheckException
+import org.partiql.eval.internal.helpers.PErrors
 import org.partiql.spi.value.Datum
 
 internal class ExprError : ExprValue {
     override fun eval(env: Environment): Datum {
-        throw TypeCheckException("A warning has been converted into an error.")
+        // TODO: Add error to ExprError
+        throw PErrors.internalErrorException(IllegalStateException("A warning has been converted into an error."))
```

Comment:
probably should be updated?

Reply:
This can be a follow-up, as it would be a backwards-compatible addition to ExprError.